R/debian.R

```diff
@@ -108,6 +108,48 @@ debian_apt_get_install <- function(droplet, ...) {
 #' @param repo CRAN mirror to use.
 install_r_package <- function(droplet, package, repo = "http://cran.rstudio.com") {
   droplet_ssh(droplet,
-    sprintf("Rscript -e \"install.packages(\'%s\', repos=\'%s/\')\"", package, repo)
+              sprintf("Rscript -e \"install.packages(\'%s\', repos=\'%s/\')\"", package, repo)
   )
 }
+
+#' @rdname debian
+#' @export
+#' @param package Name of R package to install.
+#' @param repo CRAN mirror to use.
+install_github_r_package <- function(droplet, package, repo = "http://cran.rstudio.com") {
```

Comment:
lets change the repo url to `https://cloud.r-project.org/` - and can you do that for the `install_r_package` function as well?

Reply:
Done